### PR TITLE
Upgrade Jackson to 2.16.2

### PR DESCRIPTION
**Pull request by hiroyuki-sato**

### CODE REVIEW

File: gradle/libs.versions.toml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+slf4j-api = "2.0.7"
+
+validation-api = "2.0.1.Final"
+
+# See https://github.com/FasterXML/jackson/wiki/Jackson-Releases for Jackson versions.
+#
+# We choose Jackson with the latest patch release of the latest open branch.
+jackson = "2.16.2"
+
+
```
Comment (dmikurube) [inline gradle/libs.versions.toml]:
```suggestion

```

**Review by dmikurube — APPROVED** [review]
LGTM.

It's okay this time, but let me leave a recommendation for next time -- when upgrading a dependency library, it'd be nice to have `gradle.lockfile` at first before upgrade, and then do the upgrade. By this order, we can confirm which library is upgraded explicitly.

**Comment by dmikurube** [comment]
jfyi, you're one of the maintainers of this plugin now. You can merge pull requests by yourself once approved. ;)

**Comment by hiroyuki-sato** [comment]
Thanks!

**Pull request merged**